+exp/SignalsExpTest.m

```diff
@@ -116,23 +118,32 @@
       obj.CondPars.post(condStruct);
       
       % create a listener for user changing parameters in test panel GUI
-      paramChangeL = addlistener(obj.ExpTest.ParamEditor,... 
+      addlistener(obj.ETest.ParamEditor,... 
         'Changed', @(src,event) obj.userChangedParam);
       
       % get access to PTB Screen and set viewing model (to emulate the 3
       % screens in Burgess Steering Wheel Task)
-      obj.ScreenH = obj.ExpTest.ScreenH;
+      obj.ScreenH = obj.ETest.ScreenH;
       obj.Occ = vis.init(obj.ScreenH);
       
-      screenDimsCm = [20 25]; %[width_cm height_cm of real experiment screen]
-      pxW = 960/3; % 3 screens % 1280
-      pxH = 400; % 600
-      screens(1) = vis.screen([0 0 9.5], -90, screenDimsCm, [0 0 pxW pxH]); % left screen
-      screens(2) = vis.screen([0 0 10],  0 ,... 
-        screenDimsCm, [pxW 0 2*pxW pxH]); % ahead screen
-      screens(3) = vis.screen([0 0 9.5],  90,... 
-        screenDimsCm, [2*pxW  0 3*pxW pxH]); % right screen
-      obj.Occ.screens = screens;
+      if obj.ETest.ScreenView % if this flag has been set to view PTB window as single-screen
```

Comment:
It would be nice to put this sort of thing into a separate function so that other stand-alone experiments and scripts can use it